scripts/generate_test_suites.py

```diff
@@ -0,0 +1,39 @@
+import os
+import sys
+
+parts = sys.argv[1]
+
+instrumentation_tests_path = 'instrumentation-tests/src/androidTest/java/com/mapbox/navigation/instrumentation_tests'
+core_tests = os.listdir(f'{instrumentation_tests_path}/core')
+ui_tests = os.listdir(f'{instrumentation_tests_path}/ui')
+
+core_part_size = int(len(core_tests) / int(parts)) + 1
+
+suites = {}
+for index, test in enumerate(core_tests):
+	part_number = int(index / core_part_size) + 1
+	if part_number in suites:
+		suites[part_number].append(test)
+	else:
+		suites[part_number] = [test]
+
+for key, value in suites.items():
+	tests_content = ''
+	for test in value:
+		tests_content += test.replace('.kt', '::class,\n')
```

Comment:
It's possible that some classes won't have tests in them. Will it work fine in this case?

Reply:
possible, I need to check it